protractor.conf.js

```diff
@@ -26,21 +26,6 @@ const API_CONTENT_HOST = process.env.API_CONTENT_HOST || 'http://localhost:8080'
 const MAXINSTANCES = process.env.MAXINSTANCES || 1;
 const MAX_RETRIES = process.env.MAX_RETRIES || 1;
 
-function rmDir(dirPath) {
```

Comment:
Why are you removing this? 
Without it, running locally will result in a lot of leftovers

Reply:
the screenshot plugin overwrite it after the second run